improver wxcode: use low cloud instead of cloud below 1000ft asl

benfitzpatrick opened the pull request:
We specify that weather symbols (as currently configured) use cloud amount below 1000ft asl, but we actually tend to use low type cloud fraction in the suite, although we rename it to shoehorn it in - this makes it explicit and avoids the tricky renaming within the suite.

Data renamed (cube + filename rename), and I changed an error message to make it clearer when things fail.

arh89 commented:
@benfitzpatrick Any update on the status of this one? Could we resolve conflicts and get this merged? 
@PaulAbernethy Is there a scientific reason not to make this change?

PaulAbernethy commented:
No, this change is fine. We will also need to make a change in the suite to prevent the renaming of the file.

benfitzpatrick commented:
Updated and runs fine using a new CLI results directory `improver_tests_1269_wxcode`. 

I also added a speedup, since the wxcode tests are slow on master and on this branch. It was entirely due to thread contention on load - 2-5 minutes of thread in-fighting, with about 8 seconds of actual processing. Perhaps a load_cubelist issue, since the other CLI tests aren't as bad? Switching off lazy loading fixed it - these are small input files anyway.
